apps/s_server.c

```diff
@@ -1500,7 +1506,16 @@ int s_server_main(int argc, char *argv[])
         case OPT_KEYLOG_FILE:
             keylog_file = opt_arg();
             break;
-
+        case OPT_MAX_EARLY:
+            max_early_data = atoi(opt_arg());
+            if (max_early_data < 0) {
```

Comment:
I actually used 'p' originally but changed it to 'n' because I wanted to specifically allow 0 as a value. A 'p' must be > 0.